app/helpers/report_helper.rb

```diff
@@ -146,4 +146,129 @@ def reports_group_title
          :group_description => User.current_user.current_group.description}
     end
   end
+
+  def dashboard_widget_summary(widget, sb_data)
+    schedule = widget.miq_schedule
+    timezone = schedule&.run_at ? schedule.run_at[:tz] : session[:user_tz]
+    safe_join([
+                widget_basic_information(widget),
+                widget_status(widget, sb_data, timezone),
+                widget_sb_type_report(widget, sb_data),
+                widget_timer(schedule, timezone, sb_data),
+                widget_visibility(widget),
+              ])
+  end
+
+  private
+
+  def summary_data(title)
+    {:title => _(title), :mode => "miq_widget_#{title.downcase.tr(' ', '_')}"}
```

Comment:
+1 on this, if the title variable coming in is a hard coded string (which i see that it is below) it's better to translate each of the strings individually like Gilbert said instead of this variable